NitroxModel/DataStructures/GameLogic/SubnauticaPlayerPreferences.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using BinaryPack.Attributes;
+
+namespace NitroxModel.DataStructures.GameLogic;
+
+[Serializable, DataContract]
+public class SubnauticaPlayerPreferences
```

Comment:
Subnautica is implied

Reply:
This was named to be different from [PlayerPreference](https://github.com/SubnauticaNitrox/Nitrox/blob/master/NitroxClient/GameLogic/PlayerLogic/PlayerPreferences/PlayerPreference.cs) which contains some way different data only related to Nitrox.